### DISCUSSION TITLE
SSC323DE ...

### DISCUSSION THREAD

**Discussion by Olimatou** [linux-chenxing/linux-chenxing.org#36] [General]
Hi there,

Currently working on a SigmaStar SSC323DE SoC associated with a 25L12833F type Serial NOR flash from a Xiaomi home 360 2k pro camera.

I already dumped the firmware with flashrom from the memory and will see what it is possible to modify inside to open the system for easy access.

I would like to know what it is possible to hack next.

I also got a Siglent SDS1104X-E as oscilloscope and a SDG2042X as function generator.

Regards.

[dumped_firmware.zip](https://github.com/linux-chenxing/linux-chenxing.org/files/6585553/dumped_firmware.zip)

![IMG_20210528_140811](https://user-images.githubusercontent.com/21209267/120503887-f4eace00-c3c3-11eb-9e41-bdf6da05e793.jpg)
![IMG_20210529_120032](https://user-images.githubusercontent.com/21209267/120504431-6a569e80-c3c4-11eb-9b63-3db5507ddcdf.jpg)

**Comment by fifteenhex** [comment]
Hi,

If you could add the name of the device to the wiki that would be great.
As for hacking: If you want the camera bit to work you're stuck to using the crappy vendor kernel unless someone that has the mad skills to work out how that stuff works with almost no information comes along. :(

**Reply by Olimatou** [reply]
Hi,

My first goal is to enter the embedded OS to modify stuff inside in a persistent manner, as I have already done for some other brand cheap cameras. Maybe find holes to permit access over ssh or telnet and give access to mods for everyone if possible... Not many settings on such products... So finding backdoors should help, like cutting out cloud things and add direct streams capabilities and setups ...

Currently investigating on my new cam, only had it a few days ago and disappointed about stream quality, will post current progress as it comes... Will add infos on the device to the wiki as I dig into it...

Many other things to do IRL so will post when possible.

Lack of PC software is also a concern for me, so will certainly make a software for it after I reach my goal ^^

And if you could post a direct link to the wiki you want me to feed, that should be great.

;)

![IMG_20170606_133636](https://user-images.githubusercontent.com/21209267/120519936-7dbd3600-c3d3-11eb-90a5-3bf5602b2f79.jpg)
![unnamed](https://user-images.githubusercontent.com/21209267/120519957-86157100-c3d3-11eb-868f-737f03b49f17.png)

**Comment by nurji8git** [comment]
Hello. 4 years passed since you started working on this ip camera, but if you remember where are the UART pins for this model can you help me please. I can not find them on mine. There is no information about MSJXJ06CM in the internet.

**Reply by kirill9617** [reply]
Hi! I just got this camera working with OpenIPC (built for ssc335de, but it works).
Here is UART pinout
 ![IMG_20250422_143828~2](https://github.com/user-attachments/assets/78739abc-f4d7-47ea-8ffa-4f4bc8fb13e1)
